kernel/Utilities/GlobalIndexing.cpp

```diff
@@ -58,6 +58,74 @@ GlobalIndexing::GlobalIndexing(Base::MeshManipulatorBase *mesh, Layout layout,
     reset(mesh, layout, unknowns);
 }
 
+/**
+ * Visit the element and all its boundary parts in the canonical order
+ *
+ * @param element The element to visit
+ * @param meshDimension The dimension of the mesh
+ * @param visitor The visiting function to use
+ * @tparam CV The type of the visitor, should be equivalent to
+ *  std::function<void(const MeshEntity*)>
```

Comment:
Hmm, looking at the comment I should modify it so that it should be a type which has a `operator(const MeshEntity*)`, of which the result is discarded. But I don't know of a way to actually assert this.

Reply:
Actually I think the compilation will fail if the `operator(const MeshEntity*)` does not exist